test/test_sparse.py

```diff
@@ -58,6 +58,15 @@ def all_sparse_layouts(test_name='layout', include_strided=False):
         subtest(torch.sparse_bsc, name='SparseBSC'),
     ][(0 if include_strided else 1):])
 
+def gradcheck_semantics(test_name='gradcheck'):
+    gradcheck_sparse = functools.partial(gradcheck, masked=False)
+    gradcheck_masked = functools.partial(gradcheck, masked=True)
+    gradcheck_sparse.masked = False
```

Comment:
Do you ever use these attributes?

Reply:
yes, this is useful for conditionally skipping tests that are known to fail in non-masked semantics, see `test_gradcheck_to_dense`, for instance.